timezone/src/modules/Timezone.rb

```diff
@@ -984,6 +984,12 @@ def Summary
       HTML.List(ret)
     end
 
+    # Checks whether the system has Windows installed
+    def system_has_windows?
+      win_partitions = disk_analyzer.windows_partitions.values.flatten
```

Comment:
windows_partitions returns `Hash{String => Array<::Storage::Partition>}`, so no `flatten` should be needed, right?